src/common/Parameter.cpp

```diff
@@ -5299,4 +5311,20 @@ float Parameter::calculate_modulation_value_from_string(const std::string &s, st
     return 0.0;
 }
 
+bool Parameter::supports_tuning_value_from_string(const std::string &s, std::string &errMsg)
+{
+    if (!storage)
+    {
+        // Well that's odd! But let it go.
+        return true;
+    }
+    if (storage->tuningApplicationMode == SurgeStorage::TuningApplicationMode::RETUNE_ALL)
+    {
+        errMsg = "No tuning typesin in tune-after mode";
```

Comment:
Maybe: "Fractional values not allowed in tuning applied after MIDI input mode!". If it fits?

Reply:
The string above is the longest that fits

Reply:
Oh but it shouldn’t be types in. Sigh